zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -120,10 +121,11 @@ object Server {
       reqHandler      = settings.app.compile(zExec, settings, ServerTimeGenerator.make)
       init            = ServerChannelInitializer(zExec, settings, reqHandler)
       serverBootstrap = new ServerBootstrap().channelFactory(channelFactory).group(eventLoopGroup)
-      _ <- ChannelFuture.asManaged(serverBootstrap.childHandler(init).bind(settings.address))
-
+      channel <- ZManaged.effect(serverBootstrap.childHandler(init).bind(settings.address))
+      _       <- ChannelFuture.asManaged(channel)
```

Comment:
why these changes?

Reply:
@tusharmath My bad. We don't really need it. Let me push a fix.